### ISSUE TITLE
Unable to reuse volumes after LDAP_TLS_ENFORCE is true

### ISSUE THREAD

**Issue by JSchlarb** [osixia/docker-openldap#92]
Hi there!

A really nice image, thanks for sharing :) 

I run into an issue with `docker-compose up -d --force-recreate --remove-orphans` and `LDAP_TLS_ENFORCE` true. On the first run all work as expected, but on the second start I see the error below in my log, because `/container/run/state/slapd-first-start-done` file does not exist on my second start (as expected). My workaround for now is, to map /container/run as volume, too. But I'm sure this will break forward compatibility.

``` 
+ ldapmodify -Y EXTERNAL -Q -H ldapi:/// -f /container/run/service/slapd/assets/config/tls/tls-enable.ldif
ldap_sasl_interactive_bind_s: Confidentiality required (13)
*** /container/run/startup/slapd failed with status 13
```

It looks like that the issue #36 also has the same issue.

See also at http://serverfault.com/a/617965

**Comment by g5pw** [comment]
`Confidentiality required` is probably related to the fact that `olcSecurity` is set to `1`. See also [this](http://www.openldap.org/lists/openldap-technical/201007/msg00241.html).

**Comment by g5pw** [comment]
Hmm.. .I just noticed, if LDAP_TLS_ENFORCE is true, replication will not be set up.

```
Add replication config...
588e86e4 conn=1028 fd=14 ACCEPT from PATH=/var/run/slapd/ldapi (PATH=/var/run/slapd/ldapi)
588e86e4 conn=1028 op=0 BIND dn="" method=163
588e86e4 conn=1028 op=0 RESULT tag=97 err=13 text=TLS confidentiality required
ldap_sasl_interactive_bind_s: Confidentiality required (13)
588e86e4 conn=1028 op=1 UNBIND
588e86e4 conn=1028 fd=14 closed
588e86e4 conn=1029 fd=14 ACCEPT from PATH=/var/run/slapd/ldapi (PATH=/var/run/slapd/ldapi)
588e86e4 conn=1029 op=0 BIND dn="" method=163
588e86e4 conn=1029 op=0 RESULT tag=97 err=13 text=TLS confidentiality required
ldap_sasl_interactive_bind_s: Confidentiality required (13)
588e86e4 conn=1029 op=1 UNBIND
588e86e4 conn=1029 fd=14 closed
```

**Comment by osixia** [comment]
fixed in 1.1.8

**Comment by kopax** [comment]
I am facing the same issue on 1.1.8

This appeared when settings LDAP_ENFORCE_TLS in my config.ldif:

```
olcSecurity: ssf=128
olcLocalSSF: 128
```

switching off LDAP_ENFORCE_TLS doesn't remove these lines
